build.fsx

```diff
@@ -219,7 +221,43 @@ Target.create "Restore" (fun _ -> DotNet.restore dotnetSimple "OpenTK.sln" |> ig
 Target.create "AssemblyInfo" (fun _ ->
     //TODO: Create and update the Directory.Build.props file with the version information taken from the releaseNotes value.
     // see https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#directorybuildprops-and-directorybuildtargets
-    Trace.traceError "Unimplemented.")
+
+    let getAssemblyInfoAttributes (projectName:string) =
+        [ AssemblyInfo.Title (projectName)
+          AssemblyInfo.Product project
+          AssemblyInfo.Description summary
+          AssemblyInfo.Version release.AssemblyVersion
+          AssemblyInfo.FileVersion release.AssemblyVersion
+          AssemblyInfo.CLSCompliant true
```

Comment:
I don't think we're technically CLS compliant anymore. This isn't really an attribute we (want) to provide, so let's set this to false. It's only really for VB.net users who aren't likely to be our target going forward.